hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -969,6 +969,11 @@
   public static final String HBASE_REGION_SPLIT_POLICY_KEY =
     "hbase.regionserver.region.split.policy";
 
+  public static final String RS_PARALLEL_GET_ENABLED = "hbase.server.parallel.get.enabled";
```

Comment:
I know why tihs has been done here because you wanted to use the Executors available but this being a region server config seems bit strange. But may be it is the easiest way?

Reply:
Yeah, I think this is the easiest way, using the org.apache.hadoop.hbase.executor.ExecutorService that already exist.